src/commands/profile/profile.ts

```diff
@@ -54,21 +55,35 @@ export class ProfileCommand extends SubCommandPluginCommand {
           // iterate through each of the configurations, prettyProfileDetails making the configuration more readable
           // as opposed to snake case
           // need to cast val to string since addField does not take in numbers
-          profileDisplay.addField(prettyProfileDetails[key as keyof typeof prettyProfileDetails], val.toString());
+          if (key === 'about_me') {
+            // about me can be pretty long, so we do not inline it
+            profileDisplay.addField(prettyProfileDetails[key as keyof typeof prettyProfileDetails], val.toString());
+          } else {
+            profileDisplay.addField(
+              prettyProfileDetails[key as keyof typeof prettyProfileDetails],
+              val.toString(),
```

Comment:
nitpick, but you can probably pass in 
`key !== 'about_me'` 
or something into the inline param so that you don't need an if/else